src/Documentation/DocumentationInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Documentation;
+
+/**
+ * An API documentation.
+ */
+interface DocumentationInterface
+{
```

Comment:
Why not adding some public methods?

Reply:
imo this interface should be removed at some point I really added it here to minimize the impact of this refactoring!

Reply:
Because we don't need any, IMO this will be removed in the future and replaced by OpenApi directly. I need this interface for a smooth upgrade path.